llvm/lib/SYCL/InSPIRation.cpp

```diff
@@ -44,6 +45,7 @@ using namespace llvm;
 // namespace
 namespace {
 // avoid recreation of regex's we won't alter at runtime
+cl::opt<bool> SYCLCLMeta("sycl-xlx-oclmd", cl::Hidden, cl::init(false));
```

Comment:
`SYCLCLMeta`?
`sycl-xlx-oclmd`?
And the old comment before makes things even more cryptic...